compiler/plc_ast/src/pre_processor.rs

```diff
@@ -189,7 +189,7 @@ fn process_global_variables(unit: &mut CompilationUnit, id_provider: &mut IdProv
                     data_type_declaration: ref_ty.unwrap_or(global_var.data_type_declaration.clone()),
                     initializer: None,
                     address: None,
-                    location: SourceLocation::internal(),
+                    location: SourceLocation::internal_with_unit(global_var.location.get_file_name()),
```

Comment:
Is it possible to just use the address location here?

Reply:
Yes, it is. I've updated the code and resulting snapshots